test/input/closeness_centrality_unweighted_functor_parallel_for.gt

```diff
@@ -0,0 +1,57 @@
+element Vertex end
+element Edge end
+
+const edges : edgeset{Edge}(Vertex, Vertex) = load ("../../test/graphs/test.el");
+const vertices : vertexset{Vertex} = edges.getVertices();
+
+const scores: vector{Vertex}(int) = 0;
+
+func updateEdge[checked_local: vector{Vertex}(int)](src : Vertex, dst : Vertex)
+     checked_local[dst] = checked_local[src] + 1;
```

Comment:
Can this have a race condition?

Reply:
I am not sure what does this "checked_local" do?